### PR TITLE
add new device asus zenfone max pro m1

### PR DESCRIPTION
**Pull request by iAboothahir**

### CODE REVIEW

**Comment by minlexx** [comment]
Ideally you should not just immediately merge MRs, maybe give some time for review..?

File: arch/arm64/boot/dts/qcom/sdm636-asus-x00td.dts [hunk 1]
```diff
@@ -0,0 +1,713 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, Alexey Minnekhanov <alexey.min@gmail.com>
```
Comment (minlexx) [inline arch/arm64/boot/dts/qcom/sdm636-asus-x00td.dts:3]: no, not me :(

**Comment by minlexx** [comment]
And why the revert of mdp5 config is needed?

**Comment by iAboothahir** [comment]
> And why the revert of mdp5 config is needed?

mpd5 config commit breaks display(like a blackscreen)
so reverting mdp5 config fixes display

**Pull request merged**